api/src/test/java/org/apache/iceberg/expressions/TestEvaluator.java

```diff
@@ -220,6 +222,24 @@ public void testNotEqual() {
 
   }
 
+  @Test
+  public void testStartsWith() {
+    StructType struct = StructType.of(required(24, "s", Types.StringType.get()));
+    Evaluator evaluator = new Evaluator(struct, startsWith("s", "abc"));
+    Assert.assertTrue("abc startsWith abc => true", evaluator.eval(TestHelpers.Row.of("abc")));
```

Comment:
Usually, the expectation in context is given with "should". That way, when it fails the message is "abc startsWith abc should be true". It's easy to read and understand the problem that way.

Reply:
Yeah. i used a similar format to what already exists in some of the other tests, but I didn't really like it here or either of them too much. It made it harder to see what part is needing truncation etc and it doesn't follow the very standard "should" type statements.

I agree we should update these to use should. 👍